lib/document/templates.js

```diff
@@ -69,6 +71,56 @@ const syntaxHighlighterFooter = (node, syntaxHighlighter, attrs) => {
   return ''
 }
 
+const mathjaxContent = (node) => {
```

Comment:
I think we should create a dedicated module ( `stem.js` or `mathjax.js`) that exports a `content` function.

Reply:
I put it into a stem.js module. MathJax might be an implementation detail and is also the name of the external dependency, therefore I opted for stem.js

Reply:
Perfect 👌